modules/Hoot/managers/folderManager.js

```diff
@@ -79,7 +79,7 @@ export default class FolderManager {
     }
 
     filterVisible(f) {
-        return JSON.parse(Hoot.context.storage( 'publicVisibility' )) || f.userId === Hoot.user().id;
+        return JSON.parse(Hoot.context.storage( 'publicVisibility' )) && f.id !== 0 || f.userId === Hoot.user().id && f.id !== 0;
```

Comment:
Just a suggestion, what if we put `f.id !== 0` at the beginning of this?
I am pretty sure that would achieve the same and I think doing so also says 'first and foremost, ignore the root folder'.

Reply:
I don't think we can do it like this because we do need root for Move Folder (which uses filterVisible too)